### PR TITLE
:zap: Include the jupyter-server-proxy for tensorboard function

### PR DESCRIPTION
**Pull request by harshad16**
- Include the jupyter-server-proxy for tensorboard function
 - Update the OWNERS of RHODS for repo
 - Update the CI tests

Co-authored-by: Vaishnavi Hire <vhire@redhat.com>
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [x] JIRA link(s): https://issues.redhat.com/browse/RHODS-2978
- [x] The Jira story is acked
- [ ] An entry has been added to the latest build document in [Build Announcements Folder](https://drive.google.com/drive/folders/1sgkK1WZgGo9CXsLizNe0GbAzVKuSKrZL).
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious)

### CODE REVIEW

**Comment by VaishnaviHire** [comment]
Tested custom build for JupyterHub and Notebook Controller:

**For JupyterHub:**

Users need to set following env variable in Jupyterlab:
```
import os
os.environ["TENSORBOARD_PROXY_URL"]="/user/<user-id>/proxy/6006/"
```
Result:
Screen Shot 2022-07-15 at 3 31 43 PM

---
**For Notebook- Controller**

Users need to set following env variable in Jupyterlab:
```
import os
os.environ["TENSORBOARD_PROXY_URL"]="/notebook/<nb-namespace>/<nb-name>/proxy/6006/"
```
Result:
Screen Shot 2022-07-15 at 4 42 16 PM

**Comment by lugi0** [comment]
@VaishnaviHire Is there a way to create these env vars automatically when the pod is being spawned?
The manual step of adding the proxy can be given if e.g. the user wants to use a non-standard port for tensorboard, but otherwise I think this should be already set

**Comment by samuelvl** [comment]
@lugi0 For now this will be documented as a manual step of the TensorFlow spawning process but definitely yes, we want to inject this variable in the UI spawner.

@VaishnaviHire Can you open a issue in the [dashboard repository](https://github.com/opendatahub-io/odh-dashboard) to track this feature?

**Comment by VaishnaviHire** [comment]
Dashboard Issue: https://github.com/opendatahub-io/odh-dashboard/issues/314

cc: @lugi0 @samuelvl

**Comment by lugi0** [comment]
@VaishnaviHire Do you know if this fix will work for the PyTorch image as well? the issue is present there too, and we explicitly mention Tensorboard in the list of provided libraries for that image

**Comment by VaishnaviHire** [comment]
> @VaishnaviHire Do you know if this fix will work for the PyTorch image as well? the issue is present there too, and we explicitly mention Tensorboard in the list of provided libraries for that image

This Dashboard [PR](https://github.com/opendatahub-io/odh-dashboard/pull/319), will set the env variable for both Tensorflow and Pytorch notebooks, however we would need to add `jupyter-server-proxy` to pytorch image as well. I will open the PR.

File: .aicoe-ci.yaml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 check:
-  - thoth-precommit
   - thoth-build
 build:
-  base-image: "quay.io/thoth-station/s2i-minimal-notebook:v0.0.6"
+  base-image: "quay.io/thoth-station/s2i-minimal-py38-notebook:v0.3.0"
```
Comment (lugi0) [inline .aicoe-ci.yaml:4]: wasn't pytorch's base image the minimal CUDA one? or is this something else?

Comment (harshad16) [reply .aicoe-ci.yaml:4]: This is just for normal build testing, this doesnt go into the final image.
It is just used for the build-check in the github PR, as you can see below.
and yes, CUDA minimal image used based in the build pipeline finally.

**Comment by lugi0** [comment]
Tested and working well, hold on merging until https://github.com/red-hat-data-services/ods-ci/pull/470 is merged

**Review by lugi0 — APPROVED** [review]
Automation is merged, this can move ahead

**Review by samuelvl — APPROVED** [review]

**Pull request merged**